frontend/src/routes/FacilityBooking/CreateBooking/index.tsx

```diff
@@ -249,7 +250,7 @@ export default function CreateBooking() {
               // TODO, Redirect to choose date calender page
             ></SelectableField>
           )}
-          {bookingStatus === BookingStatus.CONFLICT && <ConflictAlert errorType={'CONFLICT'} />}
+          <ConflictBookingModal modalOpen={modalIsOpen} setModalOpen={setmodalIsOpen} />
           <ButtonComponent
             state={ValidForm() ? 'primary' : 'secondary'}
```

Comment:
can help change ValidForm() to something like formIsValid()? also there is no need to change between primary and secondary. primary would be sufficient

Reply:
Resolved in commit 8a39ccd6e0e18c9e6e1d9c931c1355c8e549ffdb.